package.json

```diff
@@ -10,6 +10,7 @@
   "devDependencies": {
     "@babel/core": "^7.1.6",
     "@babel/preset-env": "^7.1.6",
+    "@types/chrome": "^0.0.126",
```

Comment:
These are community-provided types for Chrome extension APIs, from the DefinitelyTyped project. See https://www.npmjs.com/package/@types/chrome.